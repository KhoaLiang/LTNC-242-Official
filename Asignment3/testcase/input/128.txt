INSERT xMe_3 number
INSERT y string
ASSIGN y 'hello'
ASSIGN xMe_3 y