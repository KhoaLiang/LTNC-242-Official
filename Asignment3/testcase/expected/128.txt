TypeMismatch: ASSIGN xMe_3 y